src/operator-manual/monitor-mode/01-intro.md

```diff
@@ -0,0 +1,90 @@
+# Monitor mode
+
+When a policy is deployed on the cluster, it will start accepting,
+rejecting or mutating requests. It's a binary operation, either the
+policy is taking decisions on operations it targets, or it's not
+installed at all.
+
+In order to let users deploy a policy to a cluster, but without
+letting it take final decisions, the `monitor` mode has been included.
+
+The `monitor` mode is a way to deploy policies to the cluster in a way
+that:
+
+1. All requests that go through the policy will be accepted, as if the
+   policy didn't exist.
+1. The policy result is traced in the `policy-server`
+   normally. Details are included on whether the request would have
+   been rejected, or if a mutation would have been proposed by the
+   policy.
+1. The `policy-server` metrics are updated normally, with the mode
+   included in the metric baggage, so it's easy to filter policies by
+   mode, and focus on the ones on `monitor` mode.
+
+The `mode` is an attribute included in the `ClusterAdmissionPolicy`
+resource. If supports two modes: `monitor` and `protect`.
```

Comment:
can we mention that it will default to `protect` if `mode` is not specified?

Reply:
Done